apps/labs/posts/sympy_documentation.md

```diff
@@ -0,0 +1,358 @@
+---
+title: "Improving SymPy's Documentation"
+authors: [aaron-meurer]
+published: October 25, 2022
+description: 'SymPy's documentation has received many significant improvements over the past two years thanks to funding by the Chan Zuckerberg Initiative.'
+category: []
+---
+
+The Chan Zuckerberg Initiative (CZI) has funded
+[SymPy](https://www.sympy.org/) as part of Cycle 4 of its [Essential Open
+Source Software for Science (EOSS)](https://chanzuckerberg.com/eoss/) grant
+program. As part of this work, Aaron Meurer worked on improving the SymPy
+documentation, with a focus on writing new narritave documentation guides.
+
+SymPy is a mature project, and has over 1000 functions and classes. Most of
+these functions and classes have API reference documentation in the form of
+docstrings, but the SymPy documentation has historically been lacking in
+long-form narrative documentation to supplement these reference docs.
+
+In this post, I will go over some of the key documentation improvements that
+were made over the course of the 2-year grant period. Note that the
+documentation improvements were only one part of the CZI grant to improve
+SymPy. Other SymPy developers were funded to improve the performance of SymPy,
+and to improve its code generation capabilities. These improvements are
+discussed in other blog posts.
```

Comment:
The codegen blog post is here https://github.com/mechmotum/mechmotum.github.io/pull/138. I don't know when it will be published so we may need to just add the link to it later.